ui/components/app/confirm/info/row/row.tsx

```diff
@@ -100,11 +101,24 @@ function RowAlert({
     return null;
   }
 
+  const [alertModalVisible, setAlertModalVisible] = useState<boolean>(false);
+  const handleCloseModal = () => {
+    setAlertModalVisible(false);
+  };
+
   return (
     <>
+      {alertModalVisible && (
+        <MultipleAlertModal
+          alertKey={alertKey}
+          ownerId={alertOwnerId}
+          onFinalAcknowledgeClick={handleCloseModal}
+          onClose={handleCloseModal}
+        />
+      )}
       <InlineAlert
         onClick={() => {
-          // intentionally empty
+          setAlertModalVisible(true);
```

Comment:
Minor, but should we encourage local `handleX` methods for all handlers like this to avoid inline logic?

Reply:
Done